settlement/dymension/dymension.go

```diff
@@ -638,3 +633,25 @@ func (c *Client) pollForBatchInclusion(batchEndHeight uint64) (bool, error) {
 
 	return latestBatch.Batch.EndHeight == batchEndHeight, nil
 }
+
+func (c *Client) getLatestProposer() (string, error) {
```

Comment:
i'd rather have this func public instead of getting the last one by using height -1, but up to you...